### DISCUSSION TITLE
Flathub not updated to 2.7.0

### DISCUSSION THREAD

**Discussion by tgitnh** [keepassxreboot/keepassxc#7698] [Q&A] [answered]
As of now, the Flatpak on Flathub hasn't been updated to version 2.7.0, but is stuck with 2.6.6, despite having the 2.7.0 label on https://keepassxc.org/download/#linux. Is this on purpose or is it just taking time for Flathub "authorise" the new version?
Cheers!

**Comment by phoerious** [comment, marked as answer]
Please take that to the Flatpak maintainers.
